src/js/component.js

```diff
@@ -1066,6 +1066,19 @@ class Component {
     this.el_.blur();
   }
 
+  /**
+   * When this Component receives a keydown event which it does not process,
+   *  it passes the event to the Player for handling.
+   *
+   * @param {EventTarget~Event} event
+   *        The `keydown` event that caused this function to be called.
+   */
+  handleKeyPress(event) {
```

Comment:
so, basically every component calls `super.handleKeyPress` which eventually gets here, which in turn calls `handleKeyPress` on the player itself?

Reply:
Right.